math/dbcsr/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           mpi 1.0
+
+github.setup        cp2k dbcsr 2.5.0 v
```

Comment:
please add the appropriate "github.tarball_from"

Reply:
Default option from tags works, why do we need a non-default here?

Reply:
the "default" is there inly for historical reasons as it would be a lot of work to change everything as the checksums would change. 

"releases" is ideal as that signifies something that upstream explicitly did
"archive" is the next best option as it comes from an annotated tar file that *supposedly* cannot change anymore
"tag" is MacPorts default as (I guess) that was the only thing available at the time the "github" 
PG was designed (but I don't claim to have that historical knowledge).

Anyway, the above is my paraphrasing if a much longer and more eloquent description by @ryandesign somewhere - which I cannot find at the moment.

Reply:
I mean, I don’t have anything specifically against that, what I am coming from is that if a default works perfectly fine, why add extra code. (Why on Earth someone decided to change “tags” to “tarballs”? It sounds so silly and counter-intuitive.)

But yeah, I can add a non-default option for tarball_from, no big deal.

Reply:
@reneeotten Done.